src/main/java/org/cactoos/list/LimitedIterable.java

```diff
@@ -0,0 +1,40 @@
+package org.cactoos.list;
+
+import java.util.Iterator;
+
+/**
+ * Limited iterable.
+ *
+ * <p>This is a view of an existing iterable containing the given number of its
+ * first elements.</p>
+ *
+ * @author Dusan Rychnovsky (dusan.rychnovsky@gmail.com)
+ * @param <T> Element type
+ * @since 0.4
+ */
+public class LimitedIterable<T> implements Iterable<T> {
+  
+    private final Iterable<T> iterable;
+    private final int limitSize;
+
+    /**
+     * Ctor.
+     *
+     * @param iterable the underlying iterable
+     * @param limitSize the requested number of elements
+     */
+    public LimitedIterable(Iterable<T> iterable, int limitSize) {
```

Comment:
@dusan-rychnovsky it's better to use a different name here, in order to avoid collisions. Also, pay attention that compound names are a bad practice: http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html